Tests/Test/ResponseTest.swift

```diff
@@ -13,6 +13,20 @@ class ResponseTest: XCTestCase {
     let header200HEAD = "HTTP/1.1 200 OK\r\nContent-Length: 0\r\nContent-type: text/html\r\n\r\n"
     let header404HEAD = "HTTP/1.1 404 Not Found\r\nContent-Length: 0\r\nContent-type: text/html\r\n\r\n"
 
+    func readText() -> String {
+        let file = "Server/SwiftServer/log.txt"
```

Comment:
This is really a `filePath`, `pathToFile`, etc.

Also, it's not good to assume that the data file exists at that path. Instead, you should be using a configurable value such as the one that you'd pass with `-d` to CobSpec.